### PR TITLE
New Documentation Framework

### PR DESCRIPTION
**Pull request by ashucoder9**
Adds new framework for the documentation website.

### CODE REVIEW

**Review by martineckardt — DISMISSED** [review]

File: .github/CODEOWNERS
Comment (meaghanfitzgerald) [inline .github/CODEOWNERS:1]: ` .github/CODEOWNERS`
`.github/ISSUE_TEMPLATE/bug_report.md`
`.github/ISSUE_TEMPLATE/feature_request.md`
should all be kept

**Comment by meaghanfitzgerald** [comment]
There should also be a valid `.github/workflows/ci.yml` for CICD

**Comment by meaghanfitzgerald** [comment]
preview link for reference: https://avalanche-docs-git-new-release-ava-labs.vercel.app/

File: .github/CODEOWNERS
Comment (ashucoder9) [reply .github/CODEOWNERS:1]: added these files

**Comment by ashucoder9** [comment]
> There should also be a valid `.github/workflows/ci.yml` for CICD

I believe we won't need these files. Vercel integration will take care of the CI/CD.

**Comment by meaghanfitzgerald** [comment]
> > There should also be a valid `.github/workflows/ci.yml` for CICD
> 
> I believe we won't need these files. Vercel integration will take care of the CI/CD.

if the CI fails will users be able to view logs with descriptive errors so they can correct them?

**Comment by ashucoder9** [comment]
> > > There should also be a valid `.github/workflows/ci.yml` for CICD
> > 
> > 
> > I believe we won't need these files. Vercel integration will take care of the CI/CD.
> 
> if the CI fails will users be able to view logs with descriptive errors so they can correct them?

contributors outside the Vercel app prolly won't be able to see the logs by default (except that their build failed), they can also run it on local to check.

there is a setting which allows us to open logs to public but comes with some security concerns, so a bit cautious before disabling it: https://vercel.com/docs/projects/overview#build-logs-and-source-protection

**Comment by meaghanfitzgerald** [comment]
> > > > There should also be a valid `.github/workflows/ci.yml` for CICD
> > > 
> > > 
> > > I believe we won't need these files. Vercel integration will take care of the CI/CD.
> > 
> > 
> > if the CI fails will users be able to view logs with descriptive errors so they can correct them?
> 
> contributors outside the Vercel app prolly won't be able to see the logs by default (except that their build failed), they can also run it on local to check.
> 
> there is a setting which allows us to open logs to public but comes with some security concerns, so a bit cautious before disabling it: https://vercel.com/docs/projects/overview#build-logs-and-source-protection

definitely dont want the vercel logs to be public, but a simple workflow that runs `yarn build` and prints the output should suffice for in repo CI/CD

**Review by martineckardt — APPROVED** [review]

**Comment by shyos** [comment]
fyi; with this update, terraform scripts got removed from the repo and GCP tutorial links direct to 404 page.

https://docs.avax.network/nodes/on-third-party-services/google-cloud

**Pull request merged**